libc/docs/math/index.rst

```diff
@@ -275,12 +275,28 @@ Basic Operations
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextdownf    | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextdownl    | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextdownf128 | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
 | nexttoward   | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
 | nexttowardf  | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
 | nexttowardl  | |check| | |check| | |check| | |check| | |check| |         |         | |check| | |check| | |check| |         |         |
 +--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextup       | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextupf      | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextupl      | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
+| nextupf128   | |check| |         |         |         |         |         |         |         |         |         |         |         |
++--------------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+---------+
```

Comment:
I hadn't documented `nextup{,f,f128}` and `nextdown{,f,f128}` in #85431.

By the way, how exactly do you decide which targets to add entrypoints for? I only added entrypoints for Linux on x86-64 because that's what I develop on, but the current `nextup*` and `nextdown*` implementations should work on other targets too.

Reply:
So in general, there are several things you can try:
- For most of the generic functions for single, double, and long double, they should work everywhere.  You can simply add them in your patch, and if you cannot test other architectures directly, you can watch our bots https://lab.llvm.org/buildbot/#/builders?tags=libc and get ready to fix-forward or back out the change if they failed.
- For float128, so far we only tested that they work on x86-64, aarch64, and riscv-64, so normally you can just enable them for all 3, and watch the bots.
- Some targets such as `baremetal`, `windows`, or `macos` we actually don't have working bots right now, so it's ok to skip them.

Reply:
I see. I will enable other targets in a following PR.